### PR TITLE
RFC: Jira-CDEvents Integration

### PR DESCRIPTION
**Pull request by rjalander**
This RFC describes 
- an approach to integrate Jira with CDEvents using using [webhook-adapter](https://github.com/cdevents/webhook-adapter).
- and the list of Jira event types mapping with CDEvents

### CODE REVIEW

File: rfc/jira-translator-cdevents.md
Comment (xibz) [inline rfc/jira-translator-cdevents.md:22]: Can we prettify these JSON examples and put them in `<pre>`

Comment (rjalander) [reply rfc/jira-translator-cdevents.md]: formatted event json using links, outside the table

**Comment by rjalander** [comment]
@xibz could please approve if there are no other comments and merge.
Also please create a repository `jira-translator` under `cdevents` once this is merged.
Thank you.

**Review by xibz — APPROVED** [review]
Awesome! This looks good! Thanks for the updates @rjalander

**Comment by rjalander** [comment]
@xibz @afrittoli, could you please a create repository `jira-translator`

**Review by afrittoli — APPROVED** [review]

**Pull request merged**
